packages/website/docs/_samples/fiori/NavigationLayout/Basic/main.js

```diff
@@ -17,8 +21,22 @@ import "@ui5/webcomponents-icons/dist/background.js";
 import "@ui5/webcomponents-icons/dist/activity-assigned-to-goal.js";
 import "@ui5/webcomponents-icons/dist/action-settings.js";
 import "@ui5/webcomponents-icons/dist/chain-link.js";
+import "@ui5/webcomponents-icons/dist/document-text.js";
+import "@ui5/webcomponents-icons/dist/compare.js";
+import "@ui5/webcomponents-icons/dist/locked.js";
 
-var sidenav = document.querySelector("ui5-side-navigation");
-document.getElementById("toggle").addEventListener("click", () => {
-    sidenav.toggleAttribute("collapsed");
+document.querySelector("#startButton").addEventListener("click", function (event) {
+	nl1.sideCollapsed = !nl1.sideCollapsed;
+});
+
+document.querySelector("#sn1").addEventListener("selection-change", function (event) {
+	if (event.detail.item.getAttribute("target")) {
+		return;
+	}
+
+	const contentItems = document.querySelectorAll(".contentItem");
+	contentItems.forEach(item => {
+		item.classList.remove("contentItemVisible");
+	});
+	document.getElementById(event.detail.item.getAttribute("href").replace("#", "")).classList.add("contentItemVisible");
```

Comment:
if you choose "Sub Item 3", "Sub Item 4" or others below them - this line fails with error in console "NavigationLayout.html:186 Uncaught TypeError: Cannot read properties of null (reading 'classList')"